src/isar_turtlebot/turtlebot/taskhandlers/driveto.py

```diff
@@ -0,0 +1,48 @@
+import time
+from typing import Optional
+
+from isar_turtlebot.config import config
+from isar_turtlebot.models.turtlebot_status import Status
+from isar_turtlebot.ros_bridge.ros_bridge import RosBridge
+from isar_turtlebot.turtlebot.taskhandlers.taskhandler import TaskHandler
+from isar_turtlebot.utilities.pose_message import encode_pose_message
+from robot_interface.models.geometry.pose import Pose
+
+
+class DriveToHandler(TaskHandler):
+    def __init__(
+        self,
+        bridge: RosBridge,
+        publishing_timeout: float = config.getfloat("mission", "publishing_timeout"),
+    ) -> None:
+        self.bridge = bridge
+        self.publishing_timeout = publishing_timeout
+
+    def start(
+        self,
+        task_input: Pose,
+    ) -> None:
+
+        goal_pose: Pose = task_input
+        goal_id: Optional[str] = self.goal_id()
+
+        pose_message: dict = encode_pose_message(pose=goal_pose)
+        self.bridge.execute_task.publish(message=pose_message)
```

Comment:
Could the bridge.execute_task topic be renamed to something that more reflects that it only do drive_to?